ndn-cxx.sh

```diff
@@ -6,7 +6,7 @@ cd ndn-cxx
 
 ../checkout_gerrit.sh ndn-cxx
 
-./waf configure --debug --without-pch
+./waf configure --debug
```

Comment:
was there a reason for this? we're already testing the no-pch configuration in the regular jenkins builds